### ISSUE TITLE
Process is running, but window won't show up.

### ISSUE THREAD

**Issue by SilentPrayeCG** [ephtracy/ephtracy.github.io#158]
Hi. I have this issue since that version that was marked as "unstable", (that version with 256 max voxel size)

Starting up program causes process showing up in process list but window won't show up.
After few attempts, window eventually showing up, and instance which showed window successfully it uses more RAM, than instances which started up but window did not appear.

Usually i start up it 3 times, and close 2 "broken" instances with task manager.
One time, i remember, it didn't show up at all, regardless of how many time i started up exe.

Today i updated to latest version 0.99.5.1 and sadly experience same issue. I wasn't able to run it at all at first, until i rebooted and tried few more times. After it started up normally, it seems now starts up normally every time i start it up again, at least for now.

I have Win10 x64 May Update, Ryzen 2700x, GTX 1070, 64gb RAM.

**Comment by SilentPrayeCG** [comment]
I guess i found trigger that cause this behavior.
So if you start program from "start" menu or from task bar shortcut, it's (apparently) suppose to appear on the monitor on which task bar shortcut was located or on which "start" was opened.
And if i try to launch Magica on left monitor or right, it's behave that way, no window, 5% cpu loading. And if i launch several of them, it's not 5% per process, it's distribute those 5% on all Magica processes.
I'm not sure if it have something to do with windows position. Because I'm apparently have same problem with Topaz programs (like Gigapixel AI and etc), it's window actually appear, if i start it on left monitor (primary is central), but it's just white form with title and not responding, it also load 5% cpu. And it's also starts ok on primary display.

I wonder if someone else have this problem? It's apparently windows doing, but I'm not sure.

**Comment by SilentPrayeCG** [comment]
Today i found culprit, i began uninstall programs one by one...
I uninstalled NAHIMIC... and everything began working normal, immediately, even without reboot.
I'm speechless, sound enhancing software that accompanies realtek audio driver, make some non sound related programs not work on non primary monitor. Stop the Planet pls...
When i searched internet for clue, i found dead post on some forum. Here the link https://www.cnet.com/forums/discussions/programs-will-not-respond-when-they-open-on-second-monitor/
I made my own post, on this forum, if it will be not deleted, it's should be indexed and maybe help to someone who also have this issue. https://www.cnet.com/forums/discussions/programs-not-respond-if-they-open-on-second-monitor-solved/?ts=15930950333371
Case closed, i guess.
